src/mono/wasm/runtime/polyfills.ts

```diff
@@ -5,10 +5,7 @@ let node_url: any | undefined = undefined;
 
 export async function fetch_like(url: string): Promise<Response> {
     try {
-        if (typeof (globalThis.fetch) === "function") {
-            return globalThis.fetch(url, { credentials: "same-origin" });
-        }
-        else if (ENVIRONMENT_IS_NODE) {
+        if (ENVIRONMENT_IS_NODE) {
```

Comment:
@pavelsavara Are you ok with this change? When I added the fetch polyfill, it tried to use it to download runtime assets.